packages/ihe-gateway-sdk/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@metriport/ihe-gateway-sdk",
+  "version": "0.1.1-alpha.1",
+  "description": "SDK to interact with other IHE Gateways - by Metriport Inc.",
+  "author": "Metriport Inc. <contact@metriport.com>",
+  "homepage": "https://metriport.com/",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "exports": {
+    ".": "./dist/index.js",
+    "./client/*": "./dist/client/*.js",
+    "./models/*": "./dist/models/*.js",
+    "./common/*": "./dist/common/*.js"
```

Comment:
No `common` folder on this package right

Reply:
Ah, i forgot to remove it after moving that folders contents out to the `shared` package